### ISSUE TITLE
Fuel flow returns a negative value for (some) negative path angles

### ISSUE THREAD

**Issue by obbe79** [TUDelft-CNS-ATM/openap#1]
Hi,
I'm using your library to calculate fuel flow for A320. However for some negative values of the path angle I get negative values of the fuel flow.

This is what I do:
```
ff = FuelFlow(ac='A320', eng='CFM56-5B4')
ff.enroute(mass=60000, tas=301, alt=34200, path_angle=-2.1)

>>  -0.00615753894796707
```

Is this an issue or am I using your library in an incorrect way?

Thank you.

**Comment by junzis** [comment]
This was indeed a bug. The thrust should not become negative, but the idle thrust at the minimum.
It's fixed now in bee774d.

**Comment by obbe79** [comment]
I'm sorry but the new version now has other problems.
I'm not able to instanciate a FuelFlow object:

```
from openap import FuelFlow

ff = FuelFlow(ac='A320', eng='CFM56-5B4')
Traceback (most recent call last):

  File "<ipython-input-744-2bb947ed0cc5>", line 3, in <module>
    ff = FuelFlow(ac='A320', eng='CFM56-5B4')

  File "C:\Users\user\AppData\Local\Continuum\anaconda3\lib\site-packages\openap\fuel.py", line 29, in __init__
    self.thrust = Thrust(ac, eng)

  File "C:\Users\user\AppData\Local\Continuum\anaconda3\lib\site-packages\openap\thrust.py", line 29, in __init__
    super(Thrust, self).__init__()

TypeError: super(type, obj): obj must be an instance or subtype of type
```
